src/zigbee-migration-selection.html

```diff
@@ -13,19 +13,26 @@
   <div class="layout horizontal justified guides">
     <a href="/migrate-zigbee-zha-only">
       <div class="layout vertical center">
-        <img src="/images/network-migrate.png"" width="250"/>
+        <picture>
+          <source srcset="/images/icons/zha-logo.png" media="(prefers-color-scheme: dark)"> <img src="/images/icons/zha-logo.png">
```

Comment:
![CleanShot 2023-02-22 at 08 58 14](https://user-images.githubusercontent.com/1444314/220641550-7a92ad1e-f11a-4cda-8c89-609feb57de11.png)

This needs a dark logo too.

Reply:
I could not find a dark mode one under brands. I tried to create one by changing black to white and stored it locally. If it is ok, I could push it to the brands repo.